sirplatform.h

```diff
@@ -266,6 +296,10 @@ typedef void (*sir_once_fn)(void);
 
 # else /* __WIN__ */
 
+#  if defined(__MINGW32__) || defined(__MINGW64__)
+#   include <pthread.h>
```

Comment:
Can you move this up with the other includes for windows or will that break something too?

Reply:
I can move it.